tests/test_launch.py

```diff
@@ -69,8 +74,8 @@ def test_jupyter(run, no_user, monkeypatch, version):
     mock_open_browser.assert_called_with("http://localhost:1234/?token=TOKEN")
 
 
-@pytest.mark.parametrize("version", ["", "v2"])
-@pytest.mark.parametrize("gitconfig_exists", [True, False])
+@pytest.mark.parametrize("version", ["v2"])
+@pytest.mark.parametrize("gitconfig_exists", [False])
```

Comment:
question: What's the reason behind removing the arguments set by `parametrize` here? Or was this a development-only change that was committed?

(If we don't need these in the test, then the conditionals in the test that change behaviour according to them become redundant, and the `parametrize` itself becomes unnecessary.)